pkgs/tools/backup/borgbackup/default.nix

```diff
@@ -82,7 +82,6 @@ python.pkgs.buildPythonApplication rec {
   nativeCheckInputs = with python.pkgs; [
     e2fsprogs
     py
-    python-dateutil
```

Comment:
How did you learn about this input being unneeded?

Reply:
The [Changelog for 1.4.0a1](https://github.com/borgbackup/borg/blob/1.4.0/docs/changes.rst#version-140a1-2024-01-01) states:

> tests: remove python-dateutil dependency

Reply:
Thanks.